test.py

```diff
@@ -40,665 +41,601 @@
 			)
 		},
 	template_places = {
-		# *-* CANYON *-*
-		Place(
-			"in un gigantesco canyon di terra rossa che si estende per chilometri; da qui forse riusciresti a scalarne i muri",
-			exits={
-				Exit('verso la cima, scalando le rocce', {'bordo'}),
-				Exit('lungo il canyon', {'canyon'})
+		"Canyon": {
+			"chances": 5,
+			"descriptions": [
+				"Un enorme canyon di terra rossa",
+				"Un piccolo canyon"
+			],
+			"exits": {
+				"Grotta-outer": [
+					"in un buco nella roccia",
```

Comment:
Che messaggio ha l'utente quando un personaggio arriva nella sua stanza tramite questa uscita? "X arriva da te in un buco nella roccia"?